runtime/indent/logtalk.vim

```diff
@@ -38,24 +38,27 @@ function! GetLogtalkIndent()
 	endif
 	" Check for entity opening directive on previous line
 	if pline =~ '^\s*:-\s\(object\|protocol\|category\)\ze(.*,$'
-		let ind = ind + shiftwidth()
+		let ind = ind + &sw
```

Comment:
Don't use `&sw` directly. This doesn't work properly when `'sw'` is set to 0.

Reply:
Thanks for the feedback. I will look into it.